alot/commands/globals.py

```diff
@@ -40,14 +40,14 @@
 
 @registerCommand(MODE, 'exit')
 class ExitCommand(Command):
-    """Shut down cleanly.
-
-    The _prompt variable is for internal use only, it's used to control
-    prompting to close without sending, and is used by the BufferCloseCommand
-    if settings change after yielding to the UI.
-    """
+    """shut down cleanly"""
```

Comment:
Please retain the capital `S` and `.` at the end of the first line of the docstring, we try to follow PEP 257 at least somewhat to be consistent with other python projects.

Reply:
Hm. This is not followed for almost all of the other commands, but OK, I’ll put the non-capitalized version (required for consistency of command descriptions) into the decorator instead.

Reply:
Our documentation (as you noticed) is kinda all over the place.

Reply:
By the way, is there any reason for the occasional difference between between the docstring and the `help=` argument of the decorator? (See `MoveCommand`, `ReloadCommand` in the same file.) If not, perhaps it could be fixed as well.